Code/C++/integertobinarystring.cpp

```diff
@@ -0,0 +1,42 @@
+// this is a program  to change a integer to a binary string 
+// it takes integer as an input and changes it to binary string 
```

Comment:
I mean to at least explain the algorithm of this code. Not just the problem statement.